__tests__/draft-spec.js

```diff
@@ -192,6 +193,7 @@ describe('Draft', () => {
             headers: undefined,
             reply_to: [],
             reply_to_message_id: undefined,
+            tracking: {}
```

Comment:
this test calls `testContext.draft.save().then(() => ...` and since our new code checks for the existence of `tracking` before adding it to the request body, we won't expect this test to change until we add a tracking argument to the `save` call, like: `testContext.draft.save({"opens": true})`

Reply:
now that we've passed a `tracking` object into `send()` as an argument, we should expect to see it in the request body here!